src/versions.js

```diff
@@ -1,7 +1,12 @@
 const versions = {
     'react-cm-ui': {
         package: '9.15.2',
-        components: {},
+        components: {
```

Comment:
Is this the right format for versioning? I didn't find any othe example in the dev branch.

Reply:
There should be now :-)

https://github.com/saddlebackdev/react-cm-ui/blob/92ee5295c885d9f78203da24b64264029870a7bd/src/versions.js#L4-L12

Reply:
I resolved the conflict and updated this file.  See 41d68f8.
@morethanfire can you verify it looks good now?

Reply:
I commented somewhere - probably down in my latest review - on what to do here.